nbs/tutorials/tutorial.ipynb

```diff
@@ -64,30 +64,32 @@
     "4. nbdev\n",
```

Comment:
<p>nit: Jupyter Lab should be "JupyterLab" (one word)</p><p> </p><p>reference: https://jupyter.org/try</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Ah good catch, will update</p>